tests/driver_soft_spi/Makefile

```diff
@@ -0,0 +1,14 @@
+APPLICATION = driver_soft_spi
+include ../Makefile.tests_common
+
+BOARD ?= stm32f4discovery
+
+USEMODULE += soft_spi
+
+CFLAGS += "-D SOFT_SPI_PARAMS_CUSTOM = {\
```

Comment:
This looks weird? `SOFT_SPI_PARAMS_CUSTOM` only sets the miso pin here? I would say either remove the CFLAG and rely on the params file or add a complete entry. First solution should be easiest.

Reply:
```
CFLAGS += "-D SOFT_SPI_PARAMS_CUSTOM = {\
                                           .miso_pin = GPIO_UNDEF,\
																				   .mosi_pin = (GPIO_PIN(0, 0)),\
																				   .clk_pin  = (GPIO_PIN(0, 1)),\
																		   }"
```
sets all three pins